docs/howto.rst

```diff
@@ -107,62 +107,3 @@ Then reply in the bug something like this::
 and mark the bug FIXED.
 
 That's it!
-
-
-Deploys
-=======
-
-(As of May 16th, 2019)
-
-We have two environments: stage and prod.
-
-Stage autodeploys any time we land something in the master branch. You can see
-notifications on IRC or Slack. Ask for where it is these days.
-
-To deploy to prod:
-
-1. Check out the master branch and make sure you have the latest commits.
-2. Run ``scripts/deploy_bug.py`` to get the summary and description text for the
-   deploy bug.
-3. `Create a bug <https://bugzilla.mozilla.org/enter_bug.cgi?format=__standard__&product=Socorro&component=Infra>`_
-   copying the summary and description.
-
-   Make it a P1 or a P2 depending on urgency.
-
-   Assign whoever is managing the deploy.
-4. Verify all the changes on stage. For non-trivial changes, I write a comment
-   about how I verified them and any notes I had.
-5. Needinfo any engineers who have changes on stage that need verification.
-
-Wait for needinfos to resolve.
-
-6. Tag the repository using the tag number in the bug description::
-
-       $ git tag -s NNN
-
-   I structure the comment like this::
-
-       Tagging NNN for deploy to prod
-
-       COPY-AND-PASTE-CHANGES
-
-   Then the changes show up in the releases in GitHub. For example:
-
-   https://github.com/mozilla-services/socorro/releases/tag/377
-
-7. Push the tag::
-
-       $ git push --tags REMOTE TAG
-
-8. I do ``git show NNN`` and copy the output into a comment in Bugzilla.
-
-9. Watch IRC/Slack for deploy notifications and make sure the deploy succeeded.
-
-10. Depending on what went out, watch Sentry, the logs, Datadog, and other
-    things for problems.
-
-    Datadog and Sentry will send notifications for problems.
-
-11. Once everything is good to go, comment that everything is good and note
-    anything that happened that we might want to know in the future. Then mark
-    the bug FIXED.
```

Comment:
I moved this to Mana. I want to standardize on having deploy docs, infra notes, and a runbook on pages in Mana for every project we maintain. That's all tied together. For example, the runbook for "what to do when a deploy goes south" should be with other infra things.